app/views/orga/seasons/index.html.slim

```diff
@@ -27,3 +27,5 @@ table.table.table-striped.table-bordered.table-condensed
 br
 
 = link_to 'New Season', [:new, :orga, :season], class: 'btn btn-default'
+
+= render partial: 'orga/seasons/switch' if Rails.env.development?
```

Comment:
Are you sure? Wouldn't that cause trouble in test env with the tests if somebody leaves the season setting in a time travelled phase?